Specs/DateTime/English/MergedParser.json

```diff
@@ -1221,5 +1221,29 @@
       }
     }
   ]
+},
+{
+  "Input": "Schedule during the morning",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T16:12:00"
+  },
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "morning",
+      "Type": "timerange",
+      "Value": {
+        "Timex": "TMO",

```

Comment:
Values should have a list of dictionaries, see the structure on [L1213](https://github.com/Microsoft/Recognizers-Text/pull/201/files#diff-ce4b2b22cfb1cf95edb193951e24cda6L1213)